testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/userdata/ProfileSettingsActivity.kt

```diff
@@ -31,8 +29,6 @@ class ProfileSettingsActivity : BaseActivity() {
     private lateinit var profileUrl: String
     private lateinit var profileUrlBase64: String
     private lateinit var binding: ProfileSettingsActivityBinding
-    private val coroutines = MiniAppCoroutines()
```

Comment:
Sorry, I just realized that we don't actually need it as `BaseActivity` already has a similar support. So, I cleaned up a bit. I have tested it, it's working fine.